### DISCUSSION TITLE
PDF report

### DISCUSSION THREAD

**Discussion by fredsevmartins** [Open-Systems-Pharmacology/Forum#1239] [Questions & Problems]
How can I generate the pdf simulation report in the PK-SIM v11?

**Comment by msevestre** [comment]
This was discontinued in V11. Too many changes were introduced that resulted in lots of changes for a feature that is very unstable.
We'll replace it with a md (mardown) export. From there, you'll be able to generate word, pdf etx using a 3rd party toop

**Comment by fredsevmartins** [comment]
It could be great if we have this back. Sometimes the regulators ask  for simulation reports which the modeler can not modify what was reported (input). It works as a proof
